it/test/utils/WelshMessagesSpec.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package utils
+
+import helpers.ComponentSpecBase
+
+class WelshMessagesSpec extends ComponentSpecBase {
+
+  "all english messages should have a welsh translation" in {
+    val realMessages = messagesAPI.messages
+    val englishMessages = realMessages("default")
+    val welshMessages = realMessages("cy")
+
+    val missingWelshKeys = englishMessages.keySet.filterNot(key => welshMessages.keySet(key))
```

Comment:
also this might be easy to compare Set's on this occasion; as there can be also missing English keys?
something like:
https://docs.scala-lang.org/overviews/collections-2.13/sets.html
`
xs diff ys or xs &~ ys | The set difference of xs and ys.

`